### DISCUSSION TITLE
Is there any way to remap keystrokes to send to a website using Vimium-C?

### DISCUSSION THREAD

**Discussion by yozba** [gdh1995/vimium-c#1098] [Q&A] [answered]
Is it possible to use Vimium-C to remap keystrokes to send to a website?
For example, is it possible to make it so that when I type `w`, it sends `j` to YouTube?

**Comment by gdh1995** [comment, marked as answer]
Vimium C can only "simluate" keyboard events, so though its `dispatchEvent` can do so, this simulation may be detected and then refused by some websites.

An example to use dispatchEvent is https://github.com/gdh1995/vimium-c/issues/756#issuecomment-1283308837 .
```
map <v-de_k> dispatchEvent key="k-75"
# `-de_k` means "keyup", on the contrary to the default "keydown"
run K de_k-de_k
# or you may give a full list of necessary keyboard info parameters
map <v-esc> dispatchEvent type="keydown" code="Escape" key="Escape" keyCode=27
```

**Reply by yozba** [reply]
Thank you!
I was having trouble searching properly.
It seems a bit difficult, but I'll give it a try!
